### ISSUE TITLE
Inconsistent inclusion of timestamps in logs

### ISSUE THREAD

**Issue by strk** [jamulussoftware/jamulus#3004]
Logs of client connections include timestamp while logs of servers connections (to directory) do not.
This inconsistency result in either duplication of timestamp (if you add them externally) or lack of timestamp (if you don't).

See #3001

I'd suggest making it an option to include them, or never include them (relying on external tools to timestamp them).

**Comment by pljones** [comment]
Jamulus doesn't write a log of server connections to a directory.

We can remove the log of client connections to a server and have them treated the same way, though.

**Comment by strk** [comment]
Yeah, I would simplify. I was actually surprised I couldn't run ONLY a directory w/out an associated server. If it was possible I'd have less problems with this logging inconsistency :)

**Comment by pljones** [comment]
> I was actually surprised I couldn't run ONLY a directory w/out an associated server.

Currently the minimum client count is validated to be more than zero and less than (IIRC) 150.  I think we did take a look at making zero possible but it would need additional work to just allowing it and setting the client count to 1 does discourage use as a general use server.

**Comment by pljones** [comment]
Needs someone to pick up the work.  Not a pressing issue.

**Comment by mcfnord** [comment]
MY LLM WROTE:

I went through the logging call sites on `main` (@ fa8766ad) to quantify the inconsistency this issue reports.

The timestamp is not applied by a handler — it is built into the string by the caller. `CServerLogging` prepends `CurTimeDatetoLogString()` (`serverlogging.cpp:74`, format `yyyy-MM-dd HH:mm:ss` at `:106`) and then emits the line via `qInfo()`. Every other log call site passes a raw string with no timestamp.

Counts of the raw Qt log calls in `src/` (all in `.cpp`; none in headers):

```
grep -rhoE '\bq(Info|Warning|Debug|Critical)\s*\(' src/ --include=*.cpp
-> 187 total:  qInfo 97,  qWarning 45,  qDebug 25,  qCritical 20
   (12 are on comment lines -> 175 live calls; main.cpp alone holds 77)
```

No custom `qInstallMessageHandler` is installed on desktop/server builds, so those 175 lines reach Qt's default handler with no timestamp. The only handler in the tree is `myMessageHandler` (`android/androiddebug.cpp`), compiled only under `ANDROIDDEBUG`; it adds file/line/function context but also no timestamp.

So a server operator's console interleaves timestamped `CServerLogging` lines with untimestamped `qInfo`/`qWarning` lines, and nothing in the tree enforces a shared format between the two. #1464 (inconsistent log messages) has the same root: there is no single log-emission point that formats output.

Evidence only — sharing in case the counts help scope a fix.
